Gemfile

```diff
@@ -92,6 +92,11 @@ gem "jquery-ui-rails",   "5.0.5"
 gem "js_image_paths",    "0.1.0"
 gem "js-routes",         "1.2.9"
 
+# ES6
+gem "sprockets",            ">=3.0.0"
+gem "sprockets-es6",        "0.9.2"
+gem "babel-transpiler",     "0.7.0"
```

Comment:
If we, for whatever reason, stick with Sprockets 3: `babel-transpiler` is a dependency of `sprockets-es6`. Why the manual fix?